src/components/FilterList.jsx

```diff
@@ -1,12 +1,47 @@
-import PropTypes from "prop-types";
 import React from "react";
+import PropTypes from "prop-types";
+import usePets from "../hooks/usePets";
+import { Alert } from "@baltimorecounty/dotgov-components";
 
 const FilterList = ({
+  title = "",
   as: As = React.Fragment,
   items = [],
   renderItem = props => <As {...props} />,
+  filters = [],
+  apiEndpoint,
   ...props
-}) => <As {...props}>{items.map(renderItem)}</As>;
+}) => {
+  const [{ hasError, response = [], isLoading }] = usePets(apiEndpoint);
```

Comment:
im actually think we shouldn't set a default value respones here, and rather on records, because we know that is an array. Pretty sure response is an object.

Reply:
done